mps-legacy-sync-plugin/testdata/projectWithOneEmptyModel/.mps/workspace.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That file should be added to `.gitignore`.

Reply:
Done